CONTRIBUTING.md

```diff
@@ -216,9 +216,11 @@ Please update the resulting commit message, if needed. It should read as a
 coherent message. In most cases, this means not just listing the interim
 commits.
 
-If you have problems with squashing or other git workflows, you can enable
-"Allow edits from maintainers" in the right-hand sidebar of the GitHub web
-interface and ask for help in the pull request.
+If your change contains a merge commit, the above workflow may not work and you
+will need to remove the merge commit first.
+
+    git fetch https://github.com/bitcoin/bitcoin  # Fetch the latest upstream commit
+    git rebase FETCH_HEAD
```

Comment:
Does a plain `rebase` work to remove merge commits? I always assumed it was much more hassle.

Reply:
Yeah, it should remove them. Obviously you'll have to solve the conflicts again (if there were any solved in the merge commit).

Though I think this general approach should work in all possible cases regardless of how messed up  the commit history is. Any other approach requires a bit more git knowledge, but there were complaints that we don't want to dive too deep in this document.

Reply:
Resolving as fixed?